extensions/wikia/PortabilityDashboard/SpecialPortabilityDashboardController.php

```diff
@@ -14,13 +15,17 @@ public function __construct() {
 	}
 
 	public function index() {
-		$model = new PortabilityDashboardModel();
-		$list = $model->getList();
 		$langFilter = $this->getVal( self::LANGUAGE_FILTER_QS_PARAM, '' );
+		$wikiUrlParam = $this->getVal( self::WIKI_URL_QS_PARAM, '' );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)